runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingOverlay.java

```diff
@@ -54,6 +56,21 @@
 
 	private final PanelComponent panelComponent = new PanelComponent();
 
+	private final ImmutableSet<Integer> fishingAnimations = new ImmutableSet.Builder<Integer>()
+			.add(AnimationID.FISHING_BARBTAIL_HARPOON)
+			.add(AnimationID.FISHING_BAREHAND)
```

Comment:
This animation is uninterrupted, however a different animation plays when a player has caught a fish bare-handed, and that animation is missing here.

Reply:
I found four new undocumented animations (6703, 6704 upon start, 6710, 6711 upon catch) whilst barehanded fishing for Tuna. My Fishing level is 63. Can anyone with a higher fishing level let me know what the animation IDs are for Swordfish and Sharks?
To find I'd just throw `if (client.getLocalPlayer().getAnimation() != -1) log.info("Animation: {}", client.getLocalPlayer().getAnimation());` at the top of `onGameTick()`